tests/maps/loop/tiles.yaml

```diff
@@ -2,7 +2,7 @@ tiles:
     map_0/tile_0_0:
         i: 0
         j: 0
-        type: curve_left
+        type: curve
         orientation: N
```

Comment:
`orientation` should not be a field anymore, the orientation is only defined in the `frames` layer.

Reply:
Ok, so i removed orientation in tile layer